AccountingSystem/Service/ReportGeneratorHandler.py

```diff
@@ -50,15 +50,16 @@ def initializeHandler(cls, serviceInfo):
     mkDir(dataPath)
     try:
       testFile = "%s/acc.jarl.test" % dataPath
-      fd = file(testFile, "w")
+      fd = open(testFile, "w")
```

Comment:
```suggestion
      with open(testFile, "w"):
        pass
```
And delete the next line